src/rendezvous/RendezvousChannelAlgorithm.ts

```diff
@@ -0,0 +1,19 @@
+/*
+Copyright 2022 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+export enum RendezvousChannelAlgorithm {
+  ECDH_V1 = "org.matrix.msc3903.rendezvous.v1.curve25519-aes-sha256"
```

Comment:
This should be an UnstableValue, to aid the future where the client has to be able to send stable but handle both

Reply:
Done.